### PR TITLE
minor improvements and fixes

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: src/elements/map-block.html [hunk 1]
```diff
@@ -206,13 +207,28 @@
...
 
         observers: [
-          'setImages(app.data.location.images)'
+          'setImages(app.data.location.images)',
+          'pageChanged(app.page)'
         ],
 
+        pageChanged: function(page){
+
+          if (page !== 'home') {
```
Comment (ozasadnyy) [inline src/elements/map-block.html]: It's bad to hardcode route. Pass **selected** property from home page instead

File: src/elements/map-block.html [hunk 2]
```diff
@@ -114,12 +114,13 @@
...
     </style>
 
-    <google-map zoom="[[app.data.location.pointer.zoom]]" latitude="[[app.data.location.mapCenter.latitude]]"
-                longitude="[[app.data.location.mapCenter.longitude]]" api-key="[[app.data.location.apiKey]]" styles="[[options]]" disable-zoom
-                disable-default-ui>
-      <google-map-marker latitude="[[app.data.location.pointer.latitude]]"
-                         longitude="[[app.data.location.pointer.longitude]]"></google-map-marker>
-    </google-map>
+    <template is="dom-if" if="[[_active]]">
+      <google-map id="bbb" zoom="[[app.data.location.pointer.zoom]]" latitude="[[app.data.location.mapCenter.latitude]]"
```
Comment (ozasadnyy) [inline src/elements/map-block.html]: Use meaningful ids

Comment (sophieH29) [inline src/elements/map-block.html]: oops, this id shouldn't be there. I'll remove :)

**Comment by ozasadnyy** [comment]
Please make cleaner PR without unnecessary commits. PR per feature.
In any case, great work!

**Pull request closed without merging**